### PR TITLE
Resolve undefined variable in `find` block

### LINKED ISSUE #51: Undefined variable in FIND
need to throw exception for undefined variable #50 
May need to fix this after we have our dependency

### PR DESCRIPTION
**Pull request by nnhoang**
fix #51 / #50

### CODE REVIEW

**Comment by nnhoang** [comment]
@jzacsh I think I duplicated some of your code when creating `varmap`... could you check?

**Comment by jzacsh** [comment]
hmm.. i don't have any open pull-requests, so I don't think so. *but* I _did_ just review a pull request *you* wrote that looks nearly identical :P

File: src/tests/fail-undefined-variable.eq [hunk 1]
```diff
@@ -0,0 +1,11 @@
+TownCtx= {
+  people = {17;}
+}
+
+FarmCtx = {
```
Comment (jzacsh) [inline src/tests/fail-undefined-variable.eq]: Accidental merge?

Comment (nnhoang) [inline src/tests/fail-undefined-variable.eq]: No I this one is to fix undefined target in `find`. I forked from `nam-semant-multivar` the first 10 commits looks the same as in the other one

Comment (nnhoang) [inline src/tests/fail-undefined-variable.eq]: I thought when I merge `nam-semant-multivar` to master it'll update the file changes :question:

**Pull request merged**